platform-coop-toolkit/coop-library
# feat: only show language switcher if enabled via localization settings

## Description

If no languages are checked in the new Localization settings panel, the language switcher should not be shown. Otherwise, it should include only checked languages.

## Steps to test

1. Uncheck all languages in Localization settings, save.
2. Check front end.
3. Check some languages in Localization settings, save.
4. Check front end.

**Expected behavior:** Behavior matches description.

## Additional information

Not applicable.

## Related issues

Not applicable.
